Sources/NIOCore/AddressedEnvelope.swift

```diff
@@ -39,15 +39,32 @@ public struct AddressedEnvelope<DataType> {
         /// Details of any congestion state.
         public var ecnState: NIOExplicitCongestionNotificationState
         public var packetInfo: NIOPacketInfo?
+        /// The UDP segment size for Generic Segmentation Offload (GSO).
+        ///
+        /// When set, this enables per-message GSO, allowing the kernel to split this datagram
+        /// into multiple segments of the specified size. Only supported on Linux.
```

Comment:
Presumably this is just ignored on other platforms?

Reply:
Done.